### PR TITLE
link up call for counselors and open to main site

### PR DESCRIPTION
**Pull request by saragibby**
Includes 💅 to match adjustments made across rest of site.

Start to get the call for counselors landing pages linked up into the site without opening up to the public yet. Comments out portions of pages that link into call for counselors flow. 

THAT_Conference_-_Wisconsin_Dells__WI

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi/call-for-counselors**](https://that-website-git-sg-enable-call-for-speakers-page.that-conference.now.sh/wi/call-for-counselors) | [**…elor-selection-process**](https://that-website-git-sg-enable-call-for-speakers-page.that-conference.now.sh/wi/counselor-selection-process) |
|:-:|:-:|
|<a href="https://that-website-git-sg-enable-call-for-speakers-page.that-conference.now.sh/wi/call-for-counselors">Screenshot of /wi/call-for-counselors</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/77fX14ccz38XQmuxAajpPXaVKFE_">(view full size)</a> |<a href="https://that-website-git-sg-enable-call-for-speakers-page.that-conference.now.sh/wi/counselor-selection-process">Screenshot of /wi/counselor-selection-process</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/9GYOVjLqmNGPUmh9bhy1z4bjNhs_">(view full size)</a> |







Commit b53cfb521990f526d895796b72f4828ce5343778 (https://that-website-git-sg-enable-call-for-speakers-page.that-conference.now.sh).

**Comment by saragibby** [comment]
@astralbodies tagging you in on this one as I had to adjust the call for counselors landing and process pages a bit to the content we had ready. I also adjusted the wording on the perks to fill those in a bit more until we have graphics. See what you think, and feel free to add on some commits of any changes you see, or if easier just holla at me and I can make them :)

**Comment by astralbodies** [comment]
@saragibby I'll review this after our call!

File: components/Header/MessageBar.js [hunk 1]
```diff
@@ -20,10 +22,37 @@ const Location = styled.p`
...
+    gtag.event({
+      clientWindow: window,
+      action: 'click',
+      category: 'link button',
+      label: 'message bar link',
+    });
+  };
+
   return (
     <div className={className}>
```
Comment (astralbodies) [inline components/Header/MessageBar.js]: I'm not entirely sure this was introduced with your PR, but the background color isn't matching on the div for the message with the background bar. Intentional?

File: components/Header/MessageBar.js [hunk 2]
```diff
@@ -20,10 +22,37 @@ const Location = styled.p`
...
-      <Message>Call for Speakers starts January 1st!</Message>
+      <Message>
+        Call for Speakers starts January 1st!
```
Comment (astralbodies) [inline components/Header/MessageBar.js]: Could we do the following text instead:

> Call for Counselors (Speakers) starts January 1st!

Just trying to eliminate as many instances of "speakers" as possible except in those areas where an uninformed person would want to know WTH a "counselor" is like this banner link area.

File: components/Header/MessageBar.js (hunk 1 above)
Comment (saragibby) [reply components/Header/MessageBar.js]: Intentional, that is how it is designed. How the site now just a bit more noticeable as this make that area grow.

File: components/Header/MessageBar.js (hunk 2 above)
Comment (saragibby) [reply components/Header/MessageBar.js]: Good call! updated 🙂

File: components/CounselorSelectionProcess/TheProcess.js [hunk 3]
```diff
@@ -20,7 +20,8 @@ const TheProcess = () => {
         After the submission deadline, THAT Staff, together with a few other
         people we trust, will review all submissions and select the ones that
         think will make the most interesting mix at THAT Conference. We aim to
-        to have all Counselors confirmed by [TODO: DATE PLACEHOLDER].
+        to have all Counselors notified of the acceptance of their talks by
+        April 17, 2020
```
Comment (astralbodies) [inline components/CounselorSelectionProcess/TheProcess.js]: Missing period after the date.

File: components/shared/ContentSection.js [hunk 4]
```diff
@@ -24,10 +24,13 @@ const Container = styled.div`
 const ContainerInner = styled.div`
   margin: auto;
   max-width: ${props => (props.forForm ? '100rem' : '140rem')};
+<<<<<<< HEAD
```
Comment (astralbodies) [inline components/shared/ContentSection.js]: Merge conflict artifacts - oops

**Comment by saragibby** [comment]
That totally works! 😄 Thanks for reviewing @astralbodies!

**Pull request merged**